HWDAY4Week2/src/main/kotlin/com/kotlin/vehicle/main.kt

```diff
@@ -0,0 +1,19 @@
+package com.kotlin.vehicle
+
+fun main() {
```

Comment:
In the main function, you should call speedUp() and gearChange() in both instances